src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/CodeGeneration/DesignTimeNodeWriter.cs

```diff
@@ -24,6 +24,11 @@ public override void WriteUsingDirective(CodeRenderingContext context, UsingDire
         else
         {
             context.CodeWriter.WriteUsing(node.Content);
+
+            if (node.AppendLineDefaultAndHidden)
+            {
+                context.CodeWriter.WriteLine("#line hidden");
```

Comment:
I don't honestly know why both directives are always used together, so I just tried to maintain the previous behaviour here, where the first thing in a namespace was `#line hidden`, I've just moved that to the end of the using block. You make a good point though, and whilst none of the existing tests had the situation you describe, it might be possible. No harm changing this.

Reply:
Update and re-generated all baselines